ImageProcessing/src/BlueFramework/ImageProcessing/Color.h

```diff
@@ -333,43 +314,43 @@ struct Color<T, 3> {
 
 	// get the X-Value of XYZ from sRGB value
 	double getXYZ_X() {

```

Comment:
these function can be declared `const` as well

Reply:
Member functions have been made const